microsoftgraph/microsoft-graph-explorer-v4
# Fix: Bundle Monaco-editor package

## Overview

Fix #2638 , 
Fixes #2640 ,
Fixes #2628 


## Testing Instructions

* Check out this branch
* Load GE
* Observe the network tab and notice no cdn.jsdelivr network call made
* Observe monaco editors load and display results
